src/main/java/org/jenkinsci/plugins/workflow/graph/FlowNode.java

```diff
@@ -212,8 +239,9 @@ public String getDisplayFunctionName() {
     @Exported
     public BallColor getIconColor() {
         BallColor c = getError()!=null ? BallColor.RED : BallColor.BLUE;
-        // TODO this should probably also be _anime in case this is a step node with a body and the body is still running (try FlowGraphTable for example)
-        if (isRunning())        c = c.anime();
+        if (isActive()) {
```

Comment:
Visible in flow graph table. You can now see at a glance which blocks are still running.